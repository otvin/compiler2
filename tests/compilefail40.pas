program compilefail40(output);
{cannot compare char and enumerated types with relops}

type car = (van, sedan, taxi, scooter);
	transport = car;

var c,d:char;q:transport;

begin
	c := 'A';
	d := 'C';
	q := sedan;
	

	if c > q then
		writeln('oops!');

end.
