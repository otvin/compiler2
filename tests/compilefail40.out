Cannot compare types 'char' and 'transport' with relational operator in file:tests/compilefail40.pas line:15    column:10 
